frontend/src/routes/manage/Realm/Content/Edit/EditMode/util.tsx

```diff
@@ -89,3 +97,134 @@ export const NiceRadioOption = React.forwardRef<HTMLInputElement, NiceRadioOptio
         </label>
     ),
 );
+
+type VideoListFormFieldProps<TFieldValues extends FieldValues> = {
+    form: UseFormReturn<TFieldValues>;
+    order: VideoListOrder;
+    layout: VideoListLayout;
+    showTitle: boolean;
+    showMetadata: boolean;
+    allowOriginalOrder?: boolean;
+}
+
+export function VideoListFormFields<TFieldValues extends FieldValues>({
+    form,
+    order,
+    layout,
+    showTitle,
+    showMetadata,
+    allowOriginalOrder,
+}: VideoListFormFieldProps<TFieldValues>) {
+    const { t } = useTranslation();
+    const headingId = useId();
+    const optionProps = [
+        {
+            option: "order",
+            title: t("videolist-block.settings.new-to-old"),
+            checked: order === "NEW_TO_OLD",
+            value: "NEW_TO_OLD",
+        },
+        {
+            option: "order",
+            title: t("videolist-block.settings.old-to-new"),
+            checked: order === "OLD_TO_NEW",
+            value: "OLD_TO_NEW",
+        },
+        {
+            option: "order",
+            title: t("videolist-block.settings.a-z"),
+            checked: order === "AZ",
+            value: "AZ",
+        },
+        {
+            option: "order",
+            title: t("videolist-block.settings.z-a"),
+            checked: order === "ZA",
+            value: "ZA",
+        },
+    ];
+    if (allowOriginalOrder) {
+        optionProps.unshift({
+            option: "order",
+            title: t("videolist-block.settings.original"),
+            checked: order === "ORIGINAL",
+            value: "ORIGINAL",
+        });
+    }
+
+    return (
+        <div
+            css={{
+                display: "flex",
+                flexDirection: "row",
+                flexWrap: "wrap",
+                marginTop: 12,
+                justifyContent: "start",
+                rowGap: 24,
+                columnGap: 96,
+                [screenWidthAtMost(1000)]: {
+                    columnGap: 48,
+                },
+                [screenWidthAtMost(BREAKPOINT_SMALL)]: {
+                    flexDirection: "column",
+                    gap: 12,
+                },
+            }}
+        >
+            <div
+                role="group"
+                aria-labelledby={headingId + "-order"}
+            >
+                <Heading id={headingId + "-order"}>{t("videolist-block.settings.order")}</Heading>
+                <DisplayOptionGroup type="radio" {...{ form, optionProps }} />
+            </div>
+            <div
+                role="group"
+                aria-labelledby={headingId + "-view"}
+            >
+                <Heading id={headingId + "-view"}>{t("videolist-block.settings.layout")}</Heading>
+                <DisplayOptionGroup type="radio" {...{ form }} optionProps={[
+                    {
+                        option: "layout" as Path<TFieldValues>,
```

Comment:
Oh that's a leftover. It's not necessary if the cast is done in `Input.tsx` (seen in your next comment). But I agree it's kind of weird and I have to admit I don't really understand what is happening under the hood.

If I revert the code from `Input.tsx` I get the following error:
```
Type '{
  form: UseFormReturn<TFieldValues>;
  optionProps: {
    option: string;
    title: string;
    checked: boolean;
    value: string;
  }[];
  type: "radio";
}' is not assignable to type 'DisplayOptionGroup<TFieldValues>'.
```
which boils down to `Type 'string' is not assignable to type 'Path<TFieldValues>'`.
What seems weird to me is that I don't get this error in the `<DisplayOptionGroup>` in `EditMode/Video.tsx`.

Maybe it's easier to just show you directly or if you check out this branch and take a look at it yourself, if you really think this could lead to issues in the future.